ELLog/Destinations/LogConsoleDestination.swift

```diff
@@ -19,7 +19,7 @@ The default behavior is:
     showTimestamp = false
 */
 @objc(ELLogConsoleDestination)
-public class LogConsoleDestination: LogDestinationBase {
+open class LogConsoleDestination: LogDestinationBase {
```

Comment:
All access levels right now are just what was suggested by the migrator. These could be re-evaluated.